src/tree/mod.rs

```diff
@@ -187,4 +190,13 @@ mod test {
 
         assert!(true)
     }
+
+    #[test]
+    fn root_consistency_branch_tree() {
```

Comment:
Remove the unwraps. Add a `Result` as return value of the tests, and use `?` whenever is possible and `expect` where is not.

Reply:
Done